charts/cluster-addons/values.yaml

```diff
@@ -309,6 +309,79 @@ monitoring:
           persistence:
             enabled: true
             size: 10Gi
+
+  promtail:
+    enabled: true
+    chart:
+      repo: https://grafana.github.io/helm-charts
+      name: promtail
+      version: 6.17.0
+    release:
+      namespace: monitoring-system
+      values: {}
+
+  loki:
+    enabled: true
+    chart:
+      # The chart repository that contains the chart to use
+      repo: https://grafana.github.io/helm-charts
+      # The name of the chart to use
+      name: loki
+      # The version of the chart to use (must be an exact version)
+      version: 5.48.0
+    # The Helm values to use for the release
+    release:
+      namespace: monitoring-system
+      values:
+        # In case historical logs are required, configure fullnameOverride to
+        # ensure that the name matches the old loki-stack
+        # HelmRelease so that it correctly re-adopts the PVC
+        # fullnameOverride: loki-stack
+        loki:
+          compactor:
+            retention_enabled: true
+          limits_config:
+            retention_period: 72h
+          auth_enabled: false
+          commonConfig:
+            replication_factor: 1
+            # Specify path for new Loki to use same path in PVC
+            # as old loki-stack for reading previously logged data
+            # required in case historical logs are needed
+            # path_prefix: /var/loki/loki
+          storage:
+            type: 'filesystem'
+            filesystem:
+              chunks_directory: /var/loki/loki/chunks
+              rules_directory: /var/loki/loki/rules
```

Comment:
Is this also part of the config that's only required for keeping historical logs during upgrade?